books/Makefile

```diff
@@ -0,0 +1,44 @@
+version = v1.4
+repo = quay.io/redhattraining
+
+all: fmt build run
+
+fmt:
+	gofmt -w *.go
+
+build:
+	sudo buildah bud -t books:latest .
+
+run:
+	sudo podman run \
```

Comment:
Why sudo?
Lates podman should work when run as normal user.

Reply:
I switched it to sudo when buildah was having networking issues (egress), but will go back and troubleshoot. I believe there was a bug that is patched in the latest versions of buildah, but not available on RHEL yet. :/

Reply:
It looks like the issue I'm running into is that the distributed version of podman/buildah has a network bug when running rootless: https://github.com/containers/buildah/issues/1223. 

I guess an alternative is to vendor the dependencies.

Reply:
Changed to work rootless here: bd91e89ab163da3a9e9c23c1b1cd739b872d8b71